Analysis_Plan.jmd

```diff
@@ -0,0 +1,60 @@
+# Analysis Plan for BISC195 Final Project
+Jessica Maciuch
+
+## Analysis 1: Unique Kmer Count
+
+### Summary
+The first analysis will be finding the mean and standard deviation for unique kmer counts between SARS-CoV-2 strains. 
+This analysis could be useful to gauge the amount of genetic diversity within the samples for each variant.
+
+### Data
+Nucleotide sequences were obtained from NCBI using the following filters:
+Sequence Type: Nucleotide, Virus Lineage: SARS-Cov-2, Max Ambiguous Bases: 50, Completeness: Complete
+
+https://www.ncbi.nlm.nih.gov/labs/virus/vssi/#/virus?SeqType_s=Nucleotide&VirusLineage_ss=SARS-CoV-2,%20taxid:2697049&QualNum_i=50&Completeness_s=complete
+
+### Steps
+    1. Create a function that takes a random slice of sequences containing X number of sequences from each variant 
+    represented in the dataset.
+        + parse_fasta function can be modified to return tuple containing variant names pulled from headers and the sequences. 
```

Comment:
I'd recommend instead passing the headers that you get beck from `parse_fasta` to a new function instead. `parse_fasta` already does one thing. Generally, you want each function to have one contained task.